Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/EmulationFragment.java

```diff
@@ -389,7 +392,32 @@ public synchronized void clearSurface()
           NativeLibrary.WaitUntilDoneBooting();
         }
 
-        NativeLibrary.SurfaceDestroyed();
+        // AlertMessages while booting the core will dereference nullptr. Silently dismissing
```

Comment:
I don't understand how that would lead to a nullptr dereference. Could you clarify the comment with e.g. where we get the nullptr dereference?

Reply:
If you're referring to the same nullptr dereference as in the comment above, you should write that we can't call `WaitUntilDoneBooting` if we get an alert message while booting rather than just writing that we will dereference nullptr.

Reply:
Yes, I was referring to the same nullptr dereference.